lib/hooks/docker-hub-automated-build/help.html.haml

```diff
@@ -0,0 +1,4 @@
+  %dl
```

Comment:
Thanks, I removed indentation and force pushed the commit.
By the way, I tested this hook by following "Preview your hook" section in the README, I want to know how can I preview the help.html.haml? is it possible?
https://github.com/idobata/idobata-hooks#preview-your-hook